hl7v23/tests/parser_tests.bal

```diff
@@ -1,19 +1,16 @@
+import ballerina/lang.regexp;
```

Comment:
We can use the regex support inbuilt in the string lib.

Reply:
+1